src/main/java/pl/put/poznan/buildingInfo/model/Janitor.java

```diff
@@ -0,0 +1,89 @@
+package pl.put.poznan.buildingInfo.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
Is it needed here?

Reply:
It got autumatically added after I copied methods from other class, will take care of this.